internal/provider/resource_port_forward.go

```diff
@@ -0,0 +1,180 @@
+package provider
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/paultyng/go-unifi/unifi"
+)
+
+//	GetPortForward(site string, id string) (*unifi.PortForward, error)
+//	DeletePortForward(site string, id string) (*unifi.PortForward, error)
+//	CreatePortForward(site string, d *unifi.PortForward) (*unifi.PortForward, error)
+//	UpdatePortForward(site string, d *unifi.PortForward) (*unifi.PortForward, error)
+
+func resourcePortForward() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePortForwardCreate,
+		Read:   resourcePortForwardRead,
+		Update: resourcePortForwardUpdate,
+		Delete: resourcePortForwardDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"dst_port": {
+				Type:         schema.TypeString,
+				Required:     false,
+				Optional:     true,
+				ValidateFunc: validation.StringMatch(portForwardDstPortRegexp, "dst_port is invalid"),
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"fwd": {
```

Comment:
Just to make it obvious this is an IP address (and you'll need to change references below):
```suggestion
			"fwd_ip": {
```